Dockerfile.centos8

```diff
@@ -0,0 +1,12 @@
+FROM centos:8
+
+RUN dnf install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-8-x86_64/pgdg-redhat-repo-latest.noarch.rpm
+RUN dnf -qy module disable postgresql 
+RUN dnf install -y postgresql14-server
+RUN dnf install -y postgresql14-contrib
+USER postgres
+RUN /usr/pgsql-14/bin/initdb -D /var/lib/pgsql/14/data
+RUN echo "host all  all    0.0.0.0/0  md5" >> //var/lib/pgsql/14/data/pg_hba.conf
+RUN echo "listen_addresses='*'" >> /var/lib/pgsql/14/data/postgresql.conf
```

Comment:
@smolino unify the"RUNS" in 2-3 commands, no need to create 7 different layers here